core/src/plugin/base.ts

```diff
@@ -141,18 +124,18 @@ export interface RunResult {
   namespaceStatus?: NamespaceStatus
 }
 
-export const runResultSchema = () =>
-  joi
-    .object()
-    .unknown(true)
-    .keys({
-      success: joi.boolean().required().description("Whether the module was successfully run."),
-      exitCode: joi.number().integer().description("The exit code of the run (if applicable)."),
-      startedAt: joi.date().required().description("When the module run was started."),
-      completedAt: joi.date().required().description("When the module run was completed."),
-      log: joi.string().allow("").default("").description("The output log from the run."),
-      namespaceStatus: namespaceStatusSchema().optional(),
-    })
+export const runResultSchema = createSchema({
+  name: "run-result",
+  keys: {
+    success: joi.boolean().required().description("Whether the module was successfully run."),
```

Comment:
```suggestion
    success: joi.boolean().required().description("Whether the Run was successful."),
```
Also: We should probably override the description strings in `testResultSchema` to refer to Tests instead of Runs, right?

Reply:
This schema doesn't actually refer specifically or only to the `Run` action type. It is also used for Test actions.